sessionctx/variable/sysvar.go

```diff
@@ -1244,6 +1244,7 @@ var defaultSysVars = []*SysVar{
 		s.TMPTableSize = tidbOptInt64(val, DefTiDBTmpTableMaxSize)
 		return nil
 	}},
+	{Scope: ScopeGlobal, Name: TiDBTableCacheLease, Value: strconv.Itoa(DefTiDBTableCacheLease), Type: TypeUnsigned, MinValue: 1, MaxValue: 10},
```

Comment:
Even though you havee code in sysvar_cache, the convention is to add a SetGlobal func so that it runs syncronously on the node where it was issued. This also helps it be validated in tests which don't have etcd:
```suggestion
	{Scope: ScopeGlobal, Name: TiDBTableCacheLease, Value: strconv.Itoa(DefTiDBTableCacheLease), Type: TypeUnsigned, MinValue: 1, MaxValue: 10, SetGlobal: func(s *SessionVars, val string) error {	
		var val int64
		val, err = strconv.ParseInt(sVal, 10, 64)
		if err != nil {
			break
		}
		variable.TableCacheLease.Store(val)
	}},
```

In future I think we should make the sysvar_cache code obsolete and just make peers call the SetGlobal func when a value changes, but we are not there yet.